.valgrind/dlopen.supp

```diff
@@ -1,17 +1,7 @@
 {
-   Ignore dlopen bug. 
+   Ignore dlopen bug.
```

Comment:
Now that we fixed the valgrind issue, maybe reversing back to the more restrictive dlopen is better.

The new one always silence any leak for dl_open. Maybe too "violent"